packages/components/pie-radio-group/src/index.ts

```diff
@@ -179,6 +195,8 @@ export class PieRadioGroup extends FormControlMixin(RtlMixin(LitElement)) implem
 
         this.addEventListener('keydown', this._handleKeyDown, { signal });
         document.addEventListener('keydown', this._updateShiftTabState.bind(this), { signal });
+
+        this._applyNameToChildren();
```

Comment:
When we call this in `updated`, is that also called on initial page load? Or is `name` not seen as changed?

Reply:
Initial and subsequent